src/components/SecondaryPanes/CommandBar.js

```diff
@@ -235,7 +235,7 @@ class CommandBar extends Component<Props> {
       canRewind
     } = this.props;
 
-    if (canRewind || features.breakpointsDropdown) {
+    if (canRewind) {
       return;
     }
 
```

Comment:
should we just remove this stuff entirely?

Reply:
Mmmmmmmaybe?  I don't know what you mean by "this stuff".  The `canRewind` check?